### PR TITLE
Update patch release process

### PR DESCRIPTION
**Pull request by icecrime**
Ping @tiborvass @thaJeztah.

### CODE REVIEW

File: project/PATCH-RELEASES.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ review sessions take place on a weekly basis, more frequent if needed:
 - P1, P2, and P3 issues/PR's should not influence the decision, and
   should be moved to the X.Y.Z+1 milestone, or removed from the
```
Comment (tiborvass) [inline project/PATCH-RELEASES.md]: @icecrime which means that P2 and P3 issues should be moved to X.Y+1.0 milestone

Comment (icecrime) [inline project/PATCH-RELEASES.md]: That's right, formulated this way it means that we never ship P2 and P3 as part of a patch release... I need to rework this.

**Comment by icecrime** [comment]
Updated. WDYT @tiborvass?

**Comment by tiborvass** [comment]
LGTM

File: project/PATCH-RELEASES.md [hunk 2]
```diff
@@ -51,3 +51,18 @@ review sessions take place on a weekly basis, more frequent if needed:
 > a different procedure. Security releases are developed in a private
 > repository, released and tested under embargo before they become
 > publicly available.
+
+## Deciding on the content of a patch release
+
+When the criterias for moving forward with a patch release are met, the release
```
Comment (tiborvass) [inline project/PATCH-RELEASES.md]: `criteria` is plural already

File: project/PATCH-RELEASES.md [hunk 3]
```diff
@@ -51,3 +51,18 @@ review sessions take place on a weekly basis, more frequent if needed:
...
+
+- Fixes to all P0 issues *must* be included in the release.
+- Fixes to *some* P1, P2, and P3 issues *may* be included as part of the patch
+  release depending on the severity of the issue and the risk associated with
+  the patch.
+
+Any code delivered as part of a patch release should make life easier for a
+significant amount of users with zero chance of degrading anybody's experience.
+A good rule of thumb for that is to limit cherry-picking to small patches, which
+fix well understood issues, and which come with verifiable tests.
```
Comment (tiborvass) [inline project/PATCH-RELEASES.md]: `well-understood` needs a hyphen because it's an adjective.

**Comment by thaJeztah** [comment]
LGTM with Tibor's nits addressed

**Comment by icecrime** [comment]
Updated :)

**Comment by thaJeztah** [comment]
documentation CI passed, no need to run all the others. Merging

**Pull request merged**